.vscode/cspell.json

```diff
@@ -527,6 +527,8 @@
         "roups",
         "mltable",
         "wsid",
+        "myindexname",
+        "wttr",
```

Comment:
Please move this to the section that is specific for sdk/ai/azure-ai-projects/

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-python/pull/38570/commits/63823ebb97ed0454cccc170d58aee537dad2b27a)